tests/test_basic.py

```diff
@@ -18,6 +18,7 @@
 from werkzeug.routing import RequestRedirect
 
 import flask
+TEST_PORT = 8080
```

Comment:
A better approach to this solution of setting a port number with a variable is to transfer this to a configuration file. We would import that file if we need that variable and it can be reused in other test files if required.